doc-review-prompt.txt

```diff
@@ -21,7 +21,9 @@ Review Guidelines:
 - Review the document in the context of the overall user experience and functionality described.
 - Provide "reviews" ONLY if there is something to improve. Otherwise, "reviews" should be an empty array.
 - Write the review comment in the same language as the documentation.
-- For EVERY review comment of a specific line, "suggestion" MUST be the improved version of the original line. If the beginning of the original line contains Markdown syntax such as blank spaces for indentation, "-", "+", "*" for unordered list, or ">" for notes, keep them unchanged.
+- For each "suggestion", "<the improved version of the original line>" MUST keep the complete information of the original line, and then make improvements based on that. Do NOT delete any information from the original line.
+- If the original line starts with Markdown syntax (such as blank spaces for indentation, "-", "+", "*" for unordered list, or ">" for notes), keep them unchanged in "<the improved version of the original line>".
+- If the review comment is similar to "The sentence is too long, please split it into several sentences", you can directly split the content of the line into several sentences in "<the improved version of the original line>" instead of splitting it into several lines.
```

Comment:
```suggestion
- If the review comment is similar to "The sentence is too long. Please split it into several sentences", you can directly split the content of the line into several sentences in "<the improved version of the original line>" instead of splitting it into several lines.
```